_posts/2020-02-05-cockpit-212.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 212
+author: Gundersanne
+date: 2020-02-05
+tags: cockpit linux
+slug: cockpit-212
+category: release
+summary: Cockpit with documentation and time localization improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 212.
+
+### Per page documentation
+
+The help menu now includes links to page- and OS-specific documentation.
```

Comment:
... or whether the "page-" here is legit (that's how I would do it in German).

Reply:
looks fine to me.